src/index.html

```diff
@@ -15,8 +15,82 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
```

Comment:
Consider adding a fallback font for Roboto in your CSS to ensure consistent font rendering if Roboto fails to load. This can be done by specifying a generic font family like `sans-serif`.